IntroToCSharp/Pages/Activities/Variables.razor

```diff
@@ -0,0 +1,148 @@
+@inherits CaptainCoder.Page
+@page "/activities/variables"
+
+<MudText Typo="Typo.h4" Style="font-weight:bold">Variables</MudText>
+<Section Title="Introduction">
+    <MudText>An essential part of a computer program is being about to store data or “remember” it for future use. Each
+        computer has the capacity to “remember” a lot of things. Computers can remember numbers, letters, whole words,
+        and a lot more. However, they can't remember things that the programmer doesn’t explicitly tell them to. So,
+        when we
+        are coding we must use variables to label and store bits and pieces of essential data that we will need later.
+    </MudText>
+</Section>
+<Section Title="The Basics of Variables">
+    <MudText>
+        A variable is a temporary piece of stored data within the computer's memory.
```

Comment:
A variable is a label that allows us to read, store, and update data in the computer's memory.